shapefile/src/test/scala/geotrellis/shapefile/ShapeFileReaderSpec.scala

```diff
@@ -31,5 +34,17 @@ class ShapeFileReaderSpec extends AnyFunSpec with Matchers {
         data.keys.toSeq should be (Seq("LowIncome", "gbcode", "ename", "WorkingAge", "TotalPop", "Employment"))
       }
     }
+
+  }
+
+  describe("Issue 3445") {
+    it("should read SimpleFeatures with different charSet") {
+      val url = new URL("https://github.com/locationtech/geotrellis/raw/master/shapefile/data/shapefiles/demographics/demographics.shp")
```

Comment:
I'm wondering, could we put it into the `resources` folder? What is the size of it?

Reply:
Since the other test was already using the shapefile from the `shapefile/data` folder I didn't bother to mess around with it. I'm just using the URL path here because that's what the signature for `readSimpleFeatures` requires.